packages/story-editor/src/components/fontPicker/index.js

```diff
@@ -35,6 +35,7 @@ import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../constants';
 
 const FontPicker = forwardRef(function FontPicker(
   {
+    zIndex,
```

Comment:
🤔  maybe there's another way to achieve what you need here without passing _more_ zindex. This is getting us into trouble with how many layers exist.